addons/full-import/configuration.functions

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+
+function config_files() {
+  files=`perl -I/usr/local/pf/lib_perl5/lib -I/usr/local/pf/lib -Mpf::file_paths -e 'print join("\n", @pf::file_paths::stored_config_files) . "\n"'`
+  files="$files /usr/local/pf/conf/pfconfig.conf"
+  files="$files `cat add_files.txt`"
+  echo $files
+}
+
+function restore_config_files() {
+  echo "Restoring configuration files"
+  dump_dir="$1"
+  files=`config_files`
+  for f in $files; do
+    echo "Restoring $f"
+    mkdir -p `dirname $f`
+    check_code $?
+    if [ -f $dump_dir$f ]; then
+      cp -a "$dump_dir$f" $f
+      check_code $?
+      chown pf: $f
+      check_code $?
+    else
+      echo "WARN: Unable to find $f in the restore directory"
+    fi
+  done
+}
+
+function list_pf_interfaces() {
+  perl -MConfig::IniFiles -I/usr/local/pf/lib_perl/lib/perl5/ -e "print join(\"\n\", Config::IniFiles->new( -file => '/usr/local/pf/conf/pf.conf')->GroupMembers('interface')) . \"\n\"" | sed 's/^interface //g'
+}
+
+function get_os_ip_address() {
+  ip a show dev $1 | egrep -o 'inet [0-9]+\.[0-9]+\.[0-9]+\.[0-9]+' | head -1 | sed 's/^inet //g'
+}
+
+function get_os_netmask() {
+  os_ip=`ip a show dev $1 | grep inet | sed -r 's#.+inet ([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+/[0-9]+).+#\1#' | head -1`
+  ipcalc -m $os_ip | sed 's/NETMASK=//'
```

Comment:
`ipcalc` should be added has a dependency for EL and Debian.

EDIT: it looks `ipcalc` is already present on a ZEN image (CentOS7) but seems to be missing from SPEC.

`ipcalc` is packaged in bullseye, seems also ok on EL.

Reply:
ipcalc is part of the base OS, should we still add it to the spec ?

Reply:
Necessary on Debian.

Reply:
Added to both SPEC

Reply:
There is no option `-m` for `ipcalc` on Debian 11 (ipcalc 0.5, pkg: ipcalc 0.42-2)

Reply:
I'll spin up my Debian to figure out a solution